LongWarOfTheChosen/Config/XComGame.ini

```diff
@@ -102,10 +102,13 @@ DisplayName="Long War of the Chosen"
 +RequiredMods="robojumperSquadSelect_WotC"
 +RequiredMods="BetterSecondWaveSupport"
 +RequiredMods="WOTCRevertOverwatchRules"
++RequiredMods="WotCCombatKnives"
```

Comment:
I thought the only reason Why you'd make it a dependency is to not have to commit to putting experimental content in, but now? why not just integrate the assets you want in? Gives you a lot more control over things overall, and it's not that much effort either.

Reply:
I didn't want to go through the (unknown) effort of copying all the assets we need. I didn't think I could just copy the UPK, because it would conflict with any changes to the mod if someone wanted to use it with LWOTC. Having a dependency also makes it easy for us to effectively give credit to the mod author. And we benefit from improvements to the mods/animations/etc.

If it's easier than I think to integrate the assets, let me know.

Reply:
Well, unless the mod has something I don't know about it should be as easy as putting the packages into our content and changing the references. In addition, straight up integrating is better for stability, remember how the dedicated pistol mod update broke things? maybe sometimes we don't need updates.